sdk/dotnet/Apps/V1/Deployment.cs

```diff
@@ -0,0 +1,122 @@
+// *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Kubernetes.Apps.V1
+{
+    /// <summary>
+    /// Deployment enables declarative updates for Pods and ReplicaSets.
+    /// 
+    /// This resource waits until its status is ready before registering success
+    /// for create/update, and populating output properties from the current state of the resource.
+    /// The following conditions are used to determine whether the resource creation has
+    /// succeeded or failed:
+    /// 
+    /// 1. The Deployment has begun to be updated by the Deployment controller. If the current
+    ///    generation of the Deployment is &amp;gt; 1, then this means that the current generation
```

Comment:
Is `&amp;gt;` correct here?

Reply:
Yes.
<img width="596" alt="Screen Shot 2019-11-22 at 3 41 47 PM" src="https://user-images.githubusercontent.com/223467/69467849-a1afae00-0d3e-11ea-862e-56654c8c863e.png">